app/controllers/RulesController.scala

```diff
@@ -31,15 +31,18 @@ class RulesController(
       maybeRules.left.map { error => List(error.getMessage) }
     } match {
       case Right((ruleResource, lastModified)) => {
-        ruleProvisioner.updateRules(ruleResource, lastModified)
+        val ruleErrors = ruleProvisioner.updateRules(ruleResource, lastModified) match {
+          case Right(()) => Nil
+          case Left(errors) => errors.map(_.getMessage)
+        }
```

Comment:
It would be nice to prevent ingesting rules if there are issues. This could possibly be managed by rolling back to the S3 version.

Reply:
Agreed. To keep this PR focused on making existing errors visible, I'd argued that'd belong in another PR.